src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Nouns.cs

```diff
@@ -22,42 +22,47 @@ internal static class Nouns
         /// <summary>
         /// The prefix for cmdlet names.
         /// </summary>
-        private const string Prefix = "AzureRM";
+        private const string Prefix = "AzS";
```

Comment:
Is there any reason to prefer small s? Its a short form AzureStack. We have reviewed this internally multiple times